sdk/cosmos/azure-cosmos-benchmark/src/main/java/com/azure/cosmos/benchmark/ctl/AsyncCtlWorkload.java

```diff
@@ -276,13 +279,23 @@ private void createPrePopulatedDocs(int numberOfPreCreatedDocuments) {
                     PojoizedJson x =
                         resp.getItem();
                     return x;
+                }).onErrorResume(throwable -> {
+                    failureCount.incrementAndGet();
+                    logger.error("Error during pre populating item ", throwable.getMessage());
+                    return Mono.empty();
```

Comment:
We are not throwing error in case of pre-populating failures, is this intentional ?

Reply:
Yes this is intentional , this was causing ctl failure if there was single failure on pre populated document create.